app/src/main/java/com/github/se/orator/model/symblAi/AudioRecorder.kt

```diff
@@ -35,7 +34,7 @@ class AudioRecorder(
     recordingListener = listener
   }
 
-  fun startRecording() {
+  fun startRecording(audioFile: File = File(context.cacheDir, "audio_record.wav")) {
```

Comment:
I was under the impression that you mentioned it doesn't support WAV files, but from what I see, it appears to support them, which is definitely a great feature to have!

Reply:
This function is used to save files the online mode, not the offline mode.
In the offline mode I save the files as mp3 as you can see in the OfflineRecordingScreen since it's what I saw on all tutorials. I will implement the .wav soon but for now a working demo will do since that's the goal of this PR to not push too many things at once.